src/pubkey.cpp

```diff
@@ -46,7 +47,7 @@ static int ecdsa_signature_parse_der_lax(const secp256k1_context* ctx, secp256k1
     lenbyte = input[pos++];
     if (lenbyte & 0x80) {
         lenbyte -= 0x80;
-        if (pos + lenbyte > inputlen) {
+        if (lenbyte > inputlen - pos) {
             return 0;
```

Comment:
Help me understand how you believe there is a potential overflow here.

My meat based range analysis states:   Pos always has value 2 here.  Size_t lenbyte is 0-127 inclusive.  This means that the left side (which is all  size_t) has a range of 2-129.

What am I missing?